topology/University of Michigan/AGLT2/AGLT2.yaml

```
@@ -67,6 +67,39 @@ Resources:
           Monitored: true
     VOOwnership:
       ATLAS: 90
+    AGLT2-squid-3:
```

```suggestion
  AGLT2-squid-3:
```

\*shakes fist in YAML\*